imdraw/imdraw.go

```diff
@@ -51,9 +51,11 @@ type IMDraw struct {
 	Precision int
```

Comment:
And btw, if you always take the last slice from the pool, then each "layer" (external/internal/etc) will always receive the same slice, thus, if the internal layer only uses 8 vertices, its slice in the pool will be just that large and will never get bigger.